src/reducers/screen.ts

```diff
@@ -298,11 +300,15 @@ export function screen(state = initialState, action: AnyAction): ScreenState {
         }
         case types.bcAddFilter: {
             const { bcName, filter } = action.payload
+            const isDate = store.view.widgets
+                .find(item => item.name === action.payload.widgetName)
```

Comment:
What about to place `widgetName` to destructuring of `action.payload`?

Reply:
Fixed in https://github.com/tesler-platform/tesler-ui/pull/688/files